docs/community/03_governance_overview.md

```diff
@@ -6,13 +6,21 @@ title: Overview
 import Grid, { Button } from '@site/src/components/Grid'
 
 Welcome to Radworks' version-controlled governance documentation! These docs consist of carefully curated information
-and resources that lay out the Radicle’s token voting governance process and how to participate in it.
+and resources that lay out the Radworks token voting governance process and how to participate in it.
 
-Radworks’ treasury is controlled by an on-chain governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token ($RAD). Holders of this token can participate in the decision-making around the treasury by voting and creating on-chain governance proposals composed of executable actions. Currently, these actions include:
+The Radworks’ treasury is managed by an on-chain governance mechanism ([Compound-Governor](https://wiki.tally.xyz/docs/compound-governor)) via the project’s native governance token - $RAD. Holders of $RAD can participate in decision-making by voting on and creating Radworks Governance Proposals (RGPs). Voting power is derived directly from the number of tokens delegated to an address (1 token = 1 vote). 
```

Comment:
@abbey-titcomb  what do you think of adding 1 token = 1 vote at the end here?

Reply:
Yes, good idea.